src/style.css

```diff
@@ -1,3 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --black: black;
+  --blue: #00acdc;
+}
+
 body {
+  background-color: #f0f0f0;
+  width: 100%;
   margin: 0;
 }
+
+.container {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Apply `padding: 23px 0;` to `.nav__link`

Reply:
still not resolved
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/7ff9b207-a914-4bb4-9eab-834ba532ec0d)